### DISCUSSION TITLE
Awesome project, does it work with unity's WebGL?

### DISCUSSION THREAD

**Discussion by 99bits** [Iam1337/extOSC#25] [General]
Hi, 

Thank you for this amazing project.
Does it work for WebGL?

Cheers

**Comment by Iam1337** [comment]
Unfortunately, I did not do the implementation for WebGL. But since there is a demand, I will try to sketch something on this topic in August.

**Reply by 99bits** [reply]
Awesome, because more and more projects are targeting webgl as html games are increasing in popularity amongst players.
